app/presenters/tree_builder_policy_simulation_results.rb

```diff
@@ -0,0 +1,120 @@
+class TreeBuilderPolicySimulationResults < TreeBuilder
+  # exp_build_string method needed
+  include ApplicationController::ExpressionHtml
+
+  has_kids_for Hash, [:x_get_tree_hash_kids]
+
+  def initialize(name, type, sandbox, build = true, root = nil)
+    @root = root
+    super(name, type, sandbox, build)
+  end
+
+  private
+
+  def tree_init_options(_tree_name)
+    {:full_ids => true, :lazy => false}
+  end
+
+  def set_locals_for_render
+    locals = super
+    locals.merge!(:id_prefix                   => "rsop_",
+                  :autoload                    => true,
+                  :cfme_no_click               => true,
+                  :onclick                     => false,
+                  :open_close_all_on_dbl_click => true,)
+  end
+
+  def root_options
+    event = MiqEventDefinition.find(@root[:event_value])
+    [_("Policy Simulation Results for Event [%{description}]") % {:description => event.description},
+     nil,
+     "event-#{event.name}",
+     {:cfmeNoClick => true}]
+  end
+
+  def node_icon(result)
```

Comment:
`node_icon` like this is already in several TreeBuilders, can you refactor to use just one please?

(Also needs updating after #13272 , to include 100/*png in all the images.)